src/analysis/souffle/tag_transforms.dl

```diff
@@ -153,6 +155,84 @@ mustHaveIntegrityTag(result, DEFAULT_SQL_OWNER, iTag) :-
   allTagTransformPreconditionsMet(op),
   sqlPolicyRuleNameHasResult(ruleName, $AddIntegrityTag(iTag)).
 
+// In the SQL verifier, we have some merge operations that we would like to
+// propagate integrity tags in a non-default fashion (ie, rather than dropping
+// the integrity tags, we'd like them to intersect or union across all direct
+// inputs). To do that, we introduce a tag transform with special behavior:
+// rather than linking to a `SqlPolicyRule` fact, they shall instead link to
+// built-in rules here that modify how the integrity tags propagate across the
+// wrapped merge operation.
+
+#define RAKSHA_INTERNAL_INTERSECT_ITAG_RULE_NAME "##raksha_internal#intersect_itag_rule"
```

Comment:
Are the `##` used to avoid collisions?

Reply:
Yes, I just added them to make collisions less likely.